### DISCUSSION TITLE
Github org livestream

### DISCUSSION THREAD

**Discussion by EmmaDawsonDev** [EddieHubCommunity/support#4147] [General]
So I've created an org here: https://github.com/AccessibleForAll

but it's a bit of a blank slate.

What I'd like to add:
Any settings that it's good to turn on/off
How people can automatically join the org like they do with an issue on Eddiehub
People can create an issue for me to do accessibility reviews on their websites, like Eddiehub support does with portfolios/cvs etc, and auto add labels
I created a public readme but need a members/private readme
Adding tag line and extra info to the top of the org
How to create teams and add people to them eg how to add maintainers in the future

**Comment by Pradumnasaraf** [comment]
Hey! 
I think we can close this it's already done! 

Video LInk - https://www.youtube.com/watch?v=MYjdNTQhW8o

**Comment by fhrryDeveloper** [comment]
How can I help you?
